src/app-layout/visual-refresh-toolbar/skeleton/index.tsx

```diff
@@ -39,6 +39,7 @@ interface SkeletonLayoutProps
   bottomSplitPanel?: React.ReactNode;
   globalTools?: React.ReactNode;
   globalToolsOpen?: boolean;
+  navigationAnimationEnabled?: boolean;
```

Comment:
Why there is a different approach for tools and navigation panel? How is this scaling? What about split panel? Why did it not receive any fix at all?

Reply:
addressed